DOD-ISD-Verbs/dodisdverbs.jsonld

```diff
@@ -0,0 +1,4729 @@
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disposed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disposed"
+			},
+			"definition": {
+				"en": "Eliminating something or someone by throwing away, removing, or destroying."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disseminated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disseminated"
+			},
+			"definition": {
+				"en": "Spread or disperse something, especially information widely."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/drove",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "drove"
+			},
+			"definition": {
+				"en": "Propel or carry along by force in a specified direction; operate and control the direction and speed of a vehicle."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/egressed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "egressed"
+			},
+			"definition": {
+				"en": "Go out of or leave a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/elevated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "elevated"
+			},
+			"definition": {
+				"en": "Raise or lift something up to a higher position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/emplaced",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "emplaced"
+			},
+			"definition": {
+				"en": "Put in place or position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/employed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "employed"
+			},
+			"definition": {
+				"en": "Make use of something or someone; provide work to someone for payment."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/engaged",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "engaged"
+			},
+			"definition": {
+				"en": "Occupy, attract, or involve someone; enter into conflict or combat with an enemy."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/energized",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "energized"
+			},
+			"definition": {
+				"en": "Give energy, vitality and/or enthusiasm to someone or something."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/entered",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "entered"
+			},
+			"definition": {
+				"en": "Come or go into a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/established",
```

Comment:
The JSON Schema here seems incorrect -- the keys of `properties` need to be the names of the properties you want the object to have. Also, it isn't clear why this is an object when there's only going to be one property. Right now it seems to be attempting to say the extension should work something like:

```javascript
{
    "https://w3id.org/xapi/dod-isd/extensions/interactivity-level": {
        "someUnknownPropertyName": "4"
    }
}
```

If instead this should not be an object, but just a primitive value, here's an unescaped example of a JSON Schema for that:

```javascript
{
  "type": "string",
  "enum": ["red", "amber", "green"]
}
```

Also, the enumerated values are all strings, despite being numbers. It would probably be good to include documentation on what the numbers 1 to 4 mean, as well. For example, is 1 more interactive or is 4? What discriminates between the different levels?

Reply:
I agree completely and was expecting that comment. I have that descriptive data and am open to suggestions on where it should be located within the code.